apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -133,6 +134,23 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const teamHasApiV2Route = await featureRepo.checkIfTeamHasFeature(team.id, "use-api-v2-for-team-slots");
   const useApiV2 = teamHasApiV2Route && hasApiV2RouteInEnv();
 
+  const branding = getBrandingForEventType({
+    eventType: {
+      team: team.parent
+        ? {
+            ...team,
+            parent: {
+              brandColor: team.parent.brandColor,
+              darkBrandColor: team.parent.darkBrandColor,
+              theme: team.parent.theme,
+            },
+          }
+        : team,
```

Comment:
nit: I would have prefered if it could be simplified to something like. 

e.g. youy don't need to pick brandColor for team then why is it needed for team.parent
``team: team.parent ?? team``